### DISCUSSION TITLE
Add default transmogrifier to kernel + config

### DISCUSSION THREAD

**Discussion by corsairs** [TrinityCore/TrinityCore#26756] [Suggestions]
I suggest adding the default transmogrifier but not activating it.
If someone wants to install a transmogrifier, you just need to enter the command via the gm npc call.

The transmogrifier code can be taken here and slightly altered for TrinityCore.

https://github.com/azerothcore/mod-transmog

The reason why it is worth doing this.
People who cannot deliver transmogrifier install another engine and no longer participate in TrinityCore
If they don’t leave, they will help us look for new bugs.

At the moment, the current engine is a competitor of azerothcore and it has quite a few fixes.

**Comment by funjoker** [comment]
For 335 or master?
If it is 335, then it needs to be in CustomChanges.
Transmogrification is 4.3+

**Comment by Aokromes** [comment]
you have https://rochet2.github.io/Transmogrification.html for wow 3.3.5a

**Reply by corsairs** [reply]
This version is outdated and not updated.

**Reply by Faq** [reply]
try to contact him http://rochet2.github.io/contact/

**Reply by corsairs** [reply]
I think he gave up on this for a long time and is busy with something else

**Reply by ex1dreamer** [reply]
it was updated, works with recent core revs

**Reply by Faq** [reply]
Updated where? On github don't see any new commits.

**Comment by jackpoz** [comment]
https://github.com/TrinityCore/TrinityCoreCustomChanges/wiki is the place where we have custom patches that users like but that we want to keep outside of the official TC. The branches are kept up to date weekly automatically by github actions ( https://github.com/jackpoz/TrinityCoreCron/actions/workflows/trinitycore-custom-changes.yml ) , with links to the branches and to diff files.
It also provides links to patches hosted by other users.

If something is no longer maintained, we can fork it and include it in the automatic updates. The community can also request new branches by opening an issue and then a pull request.
